com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingLight.hlsl

```diff
@@ -27,7 +27,11 @@ struct LightList
 
 bool IsRectAreaLightActive(LightData lightData, float3 position, float3 normal)
 {
+#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING == 0)
```

Comment:
This is not the correct way to fix this. Going back to where "position" come from it appear it come from

#ifdef HAS_LIGHTLOOP

    // Let's compute the world space position (the non-camera relative one if camera relative rendering is enabled)
    float3 shadingPosition = GetAbsolutePositionWS(fragInput.positionRWS);

mean that if you remove "GetAbsolutePositionWS" here. It "should" work. and 
IsRectAreaLightActive should only be: float3 lightVec = position - lightData.positionRWS;

Can check woth Manu for this one

Reply:
same for other in the file

Reply:
General reules is wherever we need 'GetAbsolutePositionWS', it mean there is certainly a problem somewhere